doc/api/http.md

```diff
@@ -3284,9 +3284,10 @@ changes:
     the entire request from the client.
     See [`server.requestTimeout`][] for more information.
     **Default:** `300000`.
-  * `requireHostHeader` {boolean} It forces the server to respond with
-    a 400 (Bad Request) status code to any HTTP/1.1 request message
-    that lacks a Host header (as mandated by the specification).
+  * `requireHostHeader` {boolean} If set to `true`, it forces the server to
+    respond with a 400 (Bad Request) status code to any HTTP/1.1
+    request message that lacks a Host header
+    (as mandated by the specification).
```

Comment:
```suggestion
  * `requireHostHeader` {boolean} If set to `true`, it forces the server to
    respond with a 400 (Bad Request) status code to any HTTP/1.1
    request message that lacks a Host header
    (as mandated by the specification). **Default:** `false`.
```

Reply:
I think it's default true, its on the line right below

Reply:
Oh sorry. I missed that. I'm not why the the description was changed then. It had a default value and the required type so everything was already specified. What is the improvement of the change?

Reply:
It uses the same syntax as other fields `If set to `true`.  this was also done in a streaming to showcase the project and how to contribute so not a big deal